projects/components/src/combo-box/combo-box.component.scss

```diff
@@ -108,6 +108,7 @@
 
   .option-list {
     overflow-y: auto;
+    overflow-x: hidden;
```

Comment:
So it will clip it? Should we put ellipsis here?

Reply:
It doesn't clip in my testing. This just fixes that few pixel overflow that adds a mostly useless scrollbar.